src/components/InRequest/InRequestEditPanel.tsx

```diff
@@ -102,61 +103,19 @@ export const InRequestEditPanel: FunctionComponent<IInRequestEditPanel> = (
     reset(transRes);
   };
 
-  const updateRequest: SubmitHandler<IInRequest> = (data) => {
-    const requestApi = RequestApiConfig.getApi();
-    /* Validation has passed, so update the request */
-
-    // Transform "yes" / "no" to true/false
-    const hasExistingCAC = data?.hasExistingCAC ? true : false;
-    const isNewCivMil = data?.isNewCivMil ? true : false;
-    const isNewToBaseAndCenter = data?.isNewToBaseAndCenter ? true : false;
-    const isTraveler = data?.isTraveler ? true : false;
-
-    let dataCopy = {
-      ...data,
-      hasExistingCAC,
-      isNewCivMil,
-      isNewToBaseAndCenter,
-      isTraveler,
-    };
-
-    // If it isn't a Civ/Mil, ensure values depending on Civ/Mil only are set correctly
-    if (
-      dataCopy.empType !== EMPTYPES.Civilian &&
-      dataCopy.empType !== EMPTYPES.Military
-    ) {
-      dataCopy.isNewCivMil = false;
-      dataCopy.prevOrg = "";
-      dataCopy.isNewToBaseAndCenter = false;
-      dataCopy.isTraveler = false;
-    } else {
-      // If it is a new Civ/Mil then ensure prevOrg is set to ""
-      if (dataCopy.isNewCivMil === false) {
-        dataCopy.prevOrg = "";
-      }
-    }
-
-    if (dataCopy.empType !== EMPTYPES.Contractor) {
-      // If Employee is not a CTR then we should set hasExistingCAC to false and CACExpiration to undefined
-      dataCopy.hasExistingCAC = false;
-      dataCopy.CACExpiration = undefined;
-    } else {
-      if (!dataCopy.hasExistingCAC) {
-        // If the CTR doesn't have an Existing CAC, set the CACExpiration to undefined
-        dataCopy.CACExpiration = undefined;
-      }
-    }
```

Comment:
This logic should have been added to the RequestAPI.  It ensures certain rules for data holds true.  For instance if it is not a new Civ/Mil than prevOrg should be blank.  This will ensure "junk" data doesn't get carried over.